packages/plugins/Pets/src/SiteAdaptor/PetSetDialog.tsx

```diff
@@ -253,15 +254,19 @@ export function PetSetDialog({ configNFTs, onClose }: PetSetDialogProps) {
                     <Box component="span" className={classes.itemFix} key={option.contract}>
                         {renderImg(option)}
                         <Typography className={classes.itemTxt}>{option.name}</Typography>
-                        <Typography>{blacklist.includes(option.contract) ? t.pets_dialog_unverified() : ''}</Typography>
+                        <Typography>
+                            {blacklist.includes(option.contract) ?
+                                <Trans> (unverified)</Trans>
+                            :   ''}
+                        </Typography>
                     </Box>
                 </MenuItem>
             )}
             renderInput={(params) => (
                 <InputBase
                     {...params.InputProps}
                     fullWidth
-                    placeholder={t.pets_dialog_contract()}
+                    placeholder={_(msg`NFT Contract`)}
```

Comment:
Why don't you just use the `t` macro function?
```suggestion
                    placeholder={(t`NFT Contract`}
```

Reply:
When the language changes, `useLingui` will re-render the component, but `t` won't.

Reply:
Indeed.
But I think it's OK to use during render